enrollment-server-onboarding-common/src/main/java/com/wultra/app/onboardingserver/common/service/CommonOtpService.java

```diff
@@ -172,4 +184,30 @@ private void handleFailedOtpVerification(OnboardingProcessEntity process, OwnerI
         onboardingProcessRepository.save(process);
     }
 
+    /**
+     * Handle maximum failed attempts for OTP verification based on OTP type.
+     * @param process Onboarding process.
+     * @param otp Onboarding OTP.
+     * @param ownerId Owner identification.
+     * @return Updated onboarding process entity.
+     */
+    private OnboardingProcessEntity handleOtpMaxFailedAttempts(OnboardingProcessEntity process, OnboardingOtpEntity otp, OwnerId ownerId) {
+        if (otp.getType() == OtpType.USER_VERIFICATION) {
+            // Reset current identity verification, if possible. The process may
+            try {
+                verificationLimitService.resetIdentityVerification(ownerId);
+            } catch (RemoteCommunicationException | IdentityVerificationException | OnboardingProcessLimitException | OnboardingProcessException ex) {
+                logger.error("Identity verification reset failed, error: " + ex.getMessage(), ex);
```

Comment:
Nit: May be written as `logger.error("Identity verification reset failed, error: {}", ex.getMessage(), ex);`

Reply:
I will change it.